important-files/advantage-scope/cad/Robot_Classified/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "sourceUrl": "https://cad.onshape.com/documents/301d1509d49ed7de7df54bc7",
+  "sourceUrl": "",
```

Comment:
Why delete the URL? Be cautious when copying and pasting.

Reply:
dc1b814ea077b0ce6b97b1ce2b08288f999a9cb5
true that

Reply:
That's a good commit!